build.sbt

```diff
@@ -223,9 +225,9 @@ lazy val playSettings: Project => Project = {
       ),
       // test
       libraryDependencies ++= Seq(
-        "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % Test,
```

Comment:
There is a scala3 version: https://index.scala-lang.org/playframework/scalatestplus-play/artifacts/scalatestplus-play?binary-versions=_3&pre-releases=true